projects/storefrontapp-e2e-cypress/cypress/helpers/anonymous-consents.ts

```diff
@@ -311,6 +311,7 @@ export function showAnonymousConfigTest() {
     navigateToConsentPage();
     checkConsentsInConsentPage();
     signOutUser();
+    seeBannerAsAnonymous();
```

Comment:
Wait for the banner to ensure that the session state has reset to an anonymous user